[es]: Update translation for file glossary/compile/index-md (#9918)
mdn/translated-content: files/es/glossary/compile/index.md

> translation_of: Glossary/Compile
> ---

Reviewer: Se agrega ya que se considera el articulo actualizado completamente con la versión en inglés, este es el sha del commit del archivo en inglés

```suggestion
translation_of: Glossary/Compile
l10n:
  sourceCommit: ed947b2c608428b62a60f07d09dc543f732dc09b
---
```

La mayoría de los compiladores realizan compilación anticipada (ahead-of-time, AOT) o en tiempo de ejecución (just-in-time, JIT).

El compilador GNU `gcc` es un ejemplo muy conocido de compilador AOT. Los compiladores AOT son invocados típicamente desde la línea de comandos en un ambiente shell (desde una terminal o una consola) o dentro de un [IDE](/es/docs/Glossary/IDE).

Los compiladores JIT no son invocados típicamente de forma directa, sino que se integran internamente en software de entornos de ejecución, con el propósito de mejorar el rendimiento. Por ejemplo, la mayoría de los navegadores de ahora usan motores JavaScript que tienen compiladores JIT integrados.

> Los compiladores también pueden traducir entre lenguajes de alto nivel — por ejemplo, de TypeScript a [JavaScript](/es/docs/Glossary/JavaScript) —, en tal caso, a menudo se les denomina transpiladores.

Reviewer 2: ```suggestion
Los compiladores también pueden traducir entre lenguajes de alto nivel — por ejemplo, de TypeScript a [JavaScript](/es/docs/Glossary/JavaScript) — en tal caso, a menudo se les denomina transpiladores.
```
Las rayas son un signo de separación en sí.

> ## Saber más

Reviewer 2: ```suggestion
## Véase también
```

Usamos el véase también (ver [traducciones recomendadas](https://github.com/mdn/translated-content/tree/main/docs/es))

> - [Compilar de C/C++ a WebAssembly](/en-US/docs/WebAssembly/C_to_wasm)

Reviewer 2: ```suggestion
- [Compilar de C/C++ a WebAssembly](/es/docs/WebAssembly/C_to_wasm)
```

El fallback llevará a la versión en inglés en este caso, pero ya lo dejamos listo para cuando hagamos la traducción.

> - [Compilar de Rust a WebAssembly](/en-US/docs/WebAssembly/Rust_to_wasm)

Reviewer 2: ```suggestion
- [Compilar de Rust a WebAssembly](/es/docs/WebAssembly/Rust_to_wasm)
```

ídem anterior.
